packages/trader/src/Stores/Modules/SmartChart/Helpers/__tests__/barriers.ts

```diff
@@ -34,17 +34,17 @@ describe('Barriers', () => {
             const key_to_remove = BARRIERS_KEYS.TAKE_PROFIT;
             Barriers.removeBarrier(barriers, key_to_remove);
             expect(barriers.find(barrier => barrier.key === key_to_remove)).toBeUndefined();
-            expect(barriers.length).toEqual(2);
+            expect(barriers).toHaveLength(2);
         });
         it('should not remove any barriers if the key is not found', () => {
             Barriers.removeBarrier(barriers, BARRIERS_KEYS.STOP_LOSS);
-            expect(barriers.length).toEqual(3);
+            expect(barriers).toHaveLength(3);
```

Comment:
@kate-deriv this file doesn't seem to be related to this PR

Reply:
Hmmm, but I didn't change that file, only merge master. Will remove

Reply:
I tired to revert those test changes, but each time I tried to commit the changes my eslint corrected it back. Eslint is even underlaining that it should be changed.
Can it remain?